cnf-tests/README.md

```diff
@@ -220,6 +220,65 @@ With the following command, the local path is mounted in `/kubeconfig` inside th
     docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig quay.io/openshift-kni/cnf-tests /usr/bin/mirror --registry "my.local.registry:5000/" --images "/kubeconfig/images.json" |  oc image mirror -f -
 ```
 
+## Running tests in discovery mode
+
+The tests need to perform an environment configuration every time they are executed. This involves items such as creating Sriov Node Policies, Performance Profiles or PtpProfiles. 
+Repeating these configuration items could negatively affect the stability of the test environment.
+Discovery mode allows to configure the test environment once, and reuse this configuration for all tests in a test session, and also accros different test sessions. The tests will attempt to find configuration items needed, and use those items to execute the tests. If resources needed to run a specific test are not found, the test will be skipped (providing an appropriate message to the user).
+After the tests are finished, no cleanup of the preconfigured configuration items is done, and the test environment can immediately be used for another test run.
+
+Some configuration items are still created by the tests. These are specific items needed for a test to run, such as for example a Sriov Network. These configuration items however were chosen to include only such which can safely be create and removed without adversly affecting environment stability.
```

Comment:
Let's mention that they are created in custom namespaces that are then cleaned up. I would remove the second part, it seems something we did on resource basis, while the real driver was having namespaced resources in the test namespace that we could remove. I know SRIOV netowrk is not really in the test namespace, but the effect is in the test namespace.

Reply:
Done